main.go

```diff
@@ -50,11 +50,12 @@ func main() {
 	d := parseDuration(args["--tick"].(string))
 	max := parseInt(args["--max"].(string))
 	tmpl := readAll(os.Stdin)
-	tick := time.Tick(d)
+	ticker := time.NewTicker(d)
```

Comment:
wouldn't this panic if i pass in `--tick 0`, IIRC `NewTicker()` expects `d > 0`.